tests/OrganizationsMetadataTest.php

```diff
@@ -174,8 +179,13 @@ public function testMaintainerAdminCannotManageSystemMetadata(): void
     {
         $this->client->addUserToMaintainer($this->testMaintainerId, ['email' => $this->normalUser['email']]);
 
-        //superadmin creates system metadata
         $this->cannotManageSystemMetadata($this->normalUserClient);
```

Comment:
do tej metody by malo probudnut, cannotDeleteMetadata, takze si pred vytvorim s inym userom tie metadata a pak otestujem ze to ide listovat a ze nejde managovat system metadata

Reply:
Myslel som to tak, ze metoda `cannotManageSystemMetadata` by mala este obsahovat test ze nemoze zmazat system metadata. 

```
 public function testMaintainerAdminCannotManageSystemMetadata(): void
    {
        $this->client->addUserToMaintainer($this->testMaintainerId, ['email' => $this->normalUser['email']]);

       // superadmin creates system metadata
        $this->createSystemMetadata($this->client, $this->organization['id']);
        $metadataArray = $this->normalUserClient->listOrganizationMetadata($this->organization['id']);
       
        // metoda cannotManageSystemMetadata bude obsahovat aj test ze nejde zmazat metadata
        $this->cannotManageSystemMetadata($this->normalUserClient, $metadataArray[0]['id']);
    }
```

Reply:
Este by slo mozno lepsie pomenovat tu `cannotManageSystemMetadata` mozno nieco ako testManageSystemMetadata a ten test na `listOrganizationMetadata` by teoreticky mohol byt v tej metode

Reply:
vyreseno na callu, jen jsme prejmenovali cannotManageSystemMetadata -> cannotSetSystemMetadata a zbytek je explicitne